packages/docregistry/src/registry.ts

```diff
@@ -932,6 +932,12 @@ export namespace DocumentRegistry {
      */
     readonly ready: Promise<void>;
 
+    /**
+     * Returns whether this document context is writable. True only if the
+     * contents model is writable and collaboration is disabled.
+     */
+    readonly writable: boolean;
+
```

Comment:
Does adding this constitute a breaking API change? We need to backport this to 3.6.x. Yes, it causes TypeScript errors for plugins providing their own custom document context, but I'm not sure if this should count.